### DISCUSSION TITLE
menu music disabling

### DISCUSSION THREAD

**Discussion by pockyzx036** [ppy/osu#26782] [Ideas]
Whenever im in a main menu music is playing and even if i pause the music and start playing the game, moment i come back to the main menu, it is playing again. Im sorry if i didnt explain it properly. It's just annoyuing and should be possible to disable so u listen to music only in a map preview. Just make it more customizable. Sincerely :P

**Comment by peppy** [comment]
You can bind a key to "mute" and hit it when you want to stop the music.
